pkg/virt-config/feature-gates.go

```diff
@@ -204,3 +182,21 @@ func (config *ClusterConfig) DeclarativeHotplugVolumesEnabled() bool {
 func (config *ClusterConfig) SecureExecutionEnabled() bool {
 	return config.isFeatureGateEnabled(featuregate.SecureExecution)
 }
+
+func IsFeatureGateEnabled(featureGate string, kvConfig v1.KubeVirtConfiguration) bool {
+	if featuregate.FeatureGateInfo(featureGate).IsGA() {
+		return true
+	}
+
+	for _, fgConfig := range kvConfig.FeatureGates {
+		if fgConfig.Name == featureGate {
+			return fgConfig.IsEnabled()
```

Comment:
Do we document public API about something like
```
...
developerConfiguration:
  featureGates: 
  - "a"
featureGates:
- name: "a"
  enabled: false
...
```
? What is the expected behavior?

Reply:
> kv.spec.configuration.featureGates has precedence over kv.spec.configuration.developerConfiguration.featureGates. However, if a feature gate exists only in the legacy feature gate slice, it would be considered enabled. This way we keep backward compatibility.